### DISCUSSION TITLE
Policy Violation Audit

### DISCUSSION THREAD

**Discussion by hopenbr** [DependencyTrack/dependency-track#1158] [Q&A]
I having a hard time finding any info on what the Policy Violation Audit does.  

I cannot find any documentation on this process.  

What does it mean to approve or reject a Policy Violation?  

How would make a violation show up as audited? 

`policyViolationsSecurityUnaudited`  count does not seem to change if I set Analysis to approved/rejected

Any and all information around this would be very helpful.

**Comment by stevespringett** [comment]
> What does it mean to approve or reject a Policy Violation?

How does your organization define these things? It's the same.  DT does not define these things as they are organization specific.

> How would make a violation show up as audited?

Making an audit decision. If this is not working, then its likely a defect.

**Reply by hopenbr** [reply]
Okay, my main goal is to figure out best metrics to check for a quality gate with in a CI/CD pipeline (gitlab).  

with metrics around `policyViolationsSecurity` not working, it does not seem to be the best numbers to assess.  

Are there any commendations around this?  Maybe look at risk scores, number of critical, and high.

**Reply by roadSurfer** [reply]
Hi @hopenbr, did you get anywhere with this? I am trying to use DT withing GitLab CI/CD pipelines myself.

**Reply by hopenbr** [reply]
sorry for delay I missed this notification 

Nope, we did not choose to move forward with Dependency Track so I dropped working on it

**Reply by roadSurfer** [reply]
Thanks for getting back to me, @hopenbr.

Do you mind telling me what you used instead?
